app/src/main/java/org/shadowice/flocke/andotp/View/EntriesCardAdapter.java

```diff
@@ -119,7 +119,7 @@ public void addEntry(Entry e) {
         if (! entries.contains(e)) {
             entries.add(e);
             entriesChanged();
-            saveEntries(true);
+            saveEntries(settings.getAutoBackupEncryptedPasswordsEnabled() || settings.getAutoBackupEncryptedFullEnabled());
```

Comment:
I think this only needs ``settings.getAutoBackupEncryptedPasswordsEnabled()`` without the ``|| settings.getAutoBackupEncryptedFullEnabled()``.
The first one checks if the setting is ``!= OFF`` so it automatically includes the second one.